plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithQueryRunner.java

```diff
@@ -51,35 +53,34 @@
     private static final String ALICE_NAME = "alice";
     private static final Session ALICE = getTestSession(new Identity.Builder(ALICE_NAME).build());
 
-    private DistributedQueryRunner queryRunner;
-    private File temporaryMetastoreDirectory;
+    private static final int nodeCount = 3;
 
-    @BeforeMethod
-    public void createQueryRunner()
+    private FileHiveMetastore fileHiveMetastore;
+
+    @Override
+    protected QueryRunner createQueryRunner()
             throws Exception
     {
-        queryRunner = DistributedQueryRunner
+        DistributedQueryRunner queryRunner = DistributedQueryRunner
                 .builder(ADMIN)
-                .setNodeCount(1)
+                .setNodeCount(nodeCount)
+                // Exclude coordinator from workers to make testPartitionAppend deterministically reproduce the original problem
```

Comment:
The comment needs to be more specific now (since there are multiple tests in this class) - maybe refer to #6853 or inline a short description of the issue.

Reply:
I've mede it more explicit to point out test which requires this setting.